### PR TITLE
Add failing python-build-standalone alpine test

### PR DESCRIPTION
**Pull request by konstin**
Currently, python-build-standalone is failing on alpine (musl) with the following error:

> OSError: Dynamic loading not supported

iirc this is something that gregory mentioned, but it means that as of now, we can't adopt bootstrapping python-build-standalone as default when we want to support alpine.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -32,7 +32,7 @@ jobs:
     name: "cargo clippy"
     strategy:
       matrix:
-        os: [ubuntu-latest, windows-latest]
+        os: [ ubuntu-latest, windows-latest ]
```
Comment (zanieb) [inline .github/workflows/ci.yml:35]: What is this madness formatting? :D

Comment (konstin) [reply .github/workflows/ci.yml:35]: pycharm

Comment (konstin) [reply .github/workflows/ci.yml:35]: https://github.com/astral-sh/uv/pull/2406

Comment (konstin) [reply .github/workflows/ci.yml:35]: (I'm not fixing this PR until the underlying bug it fixed)

**Comment by zanieb** [comment]
@konstin what's the plan here?

**Comment by konstin** [comment]
We need to fix musl in python-build-standalone, then we can use this branch as a test that the problem with static vs dynamic linking is indeed fixed.

**Pull request closed without merging**